adele-booking/src/main/java/com/microservicesteam/adele/booking/domain/BookingRequest.java

```diff
@@ -13,6 +19,16 @@
 
     List<Integer> positions();
 
+    default ImmutableList<Position> requestedPositions() {
+        return positions().stream()
+                .map(positionId -> Position.builder()
+                        .eventId(eventId())
+                        .sectorId(sectorId())
+                        .id(positionId)
+                        .build())
+                .collect(collectingAndThen(toList(), ImmutableList::copyOf));
```

Comment:
Revelation of the day: Guava 21.0 `ImmutableList` has a method called `toImmutableList()`, therefore the above line could be simplified to `.collect(ImmutableList.toImmutableList());`
Side note: this is a good candidate for a KAIZEN.